make_timeline.py

```diff
@@ -69,7 +69,7 @@ def build(self):
 		self.g_axis.translate(0, y_axis)
 		self.drawing.add(self.g_axis)
 		# finally set the height on the drawing
-		self.drawing['height'] = height
+		self.drawing['height'] = height * 1.4
```

Comment:
Just out of curiosity, why did you want to increase the image height by 40%?  Rather than hardcoding the number, maybe this could be a command line argument?